### ISSUE TITLE
Checkpoint 15 issues

### ISSUE THREAD

**Issue by snychka** [ps-interactive/asp-net-core-6-fundamentals#16]
Issues:
1. ~~Task false fails~~ done!<br />
   <img width="1533" alt="Screen Shot 2022-11-04 at 10 52 27 AM" src="https://user-images.githubusercontent.com/5722849/200031790-a5a1b197-3a84-4fb7-87c4-f66b40c87538.png">

1. remove feedback (see [What are the recommended layout and settings for a Checkpoint?](https://docs.google.com/document/d/1kmko29RHzusIBqZolIJGnlzlyMBzjbm2HX-NSFnAYAI/edit#bookmark=id.w9x785qwyi8l) in the FAQ)
1. ~~remove the **Terminal** (see #1)~~

**Comment by snychka** [comment]
@GillCleeren, please @ me & assign to me when you've fixed this one, as I need to still review it post-fix

notes for me: different order, no var, use this.

**Comment by GillCleeren** [comment]
@snychka done

**Comment by snychka** [comment]
It still false failed. It was due to .csproj-based compilation errors. I fixed it, 'twas due to case-sensitivity on linux, probably meaning it worked on windows: https://github.com/ps-interactive/asp-net-core-6-fundamentals/commit/fdd4fbbd946db40b140e8cb78ad8725f2b421fe5

did a sync in Codey and Save.

Your tests worked well too, quite robust to different, but correct code! 👏 

I also changed check to look for Failed!, and changed the resources to Large, jic.

1, now, and 3 look good! 2 not, but that seems ubiquitous, I'll deal with that in one global issue
